content/en/tracing/trace_collection/compatibility/dotnet-core.md

```diff
@@ -92,9 +92,25 @@ The [latest version of the .NET Tracer][4] can automatically instrument the foll
 
 Don't see the library you're looking for? First, check if the library produces observability data compatible with OpenTelemetry (see [Using OpenTelemetry Instrumentation Libraries][13] for more details). If not, Datadog is continually adding additional support. [Check with the Datadog team][5] for help.
 
+## OpenTelemetry based integrations
```

Comment:
Didn't expect to see this either

Reply:
This must be a bad merge. This content was removed in a recent PR so it looks like added content. @brett0000FF please remove this section entirely